sysrsync/runner.py

```diff
@@ -1,18 +1,22 @@
 import os
 import subprocess
-
+import inspect
 from sysrsync.command_maker import get_rsync_command
 from sysrsync.exceptions import RsyncError
 
 
 def run(cwd=os.getcwd(), strict=True, verbose=False, **kwargs):
     rsync_command = get_rsync_command(**kwargs)
     rsync_string = ' '.join(rsync_command)
-
+    rsync_args = inspect.getfullargspec(get_rsync_command)[0]
+    subp_kwargs = {}
+    for k, v in kwargs.items():
+        if k not in rsync_args:
+            subp_kwargs[k] = v
```

Comment:
maybe to avoid the risk of colliding `get_rsync_command` and `subprocess.run` arguments, we could add a separate argument for `sysrsync.run` to pass those args to subprocess.run`?